doc/source/cluster/kubernetes/examples/rayjob-batch-inference-example.md

```diff
@@ -0,0 +1,144 @@
+
+Note the following fields in the `RayJob` spec, which specify the Ray image and the GPU resources for our Ray node:
+
+```yaml
+        spec:
+          containers:
+            - name: ray-head
+              image: rayproject/ray-ml:2.6.3-gpu
+              resources:
+                limits:
+                  nvidia.com/gpu: "4"
+                  cpu: "54"
+                  memory: "54Gi"
+                requests:
+                  nvidia.com/gpu: "4"
+                  cpu: "54"
+                  memory: "54Gi"
+              volumeMounts:
+                - mountPath: /home/ray/samples
+                  name: code-sample
+          nodeSelector:
+            cloud.google.com/gke-accelerator: nvidia-tesla-t4 # This is the GPU type we used in the GPU node pool.
+```
+
+To submit the job, run the following command:
+
+```bash
+kubectl apply -f ray-job.batch-inference.yaml
+```
+
+We can check the status with `kubectl describe rayjob rayjob-sample`.
+
+Sample output:
+
+```
+[...]
+Status:
+  Dashboard URL:          rayjob-sample-raycluster-j6t8n-head-svc.default.svc.cluster.local:8265
+  End Time:               2023-08-22T22:48:35Z
+  Job Deployment Status:  Running
+  Job Id:                 rayjob-sample-ft8lh
+  Job Status:             SUCCEEDED
+  Message:                Job finished successfully.
+  Observed Generation:    2
+  Ray Cluster Name:       rayjob-sample-raycluster-j6t8n
+  Ray Cluster Status:
+    Endpoints:
+      Client:        10001
+      Dashboard:     8265
+      Gcs - Server:  6379
+      Metrics:       8080
+    Head:
+      Pod IP:             10.112.1.3
+      Service IP:         10.116.1.93
+    Last Update Time:     2023-08-22T22:47:44Z
+    Observed Generation:  1
+    State:                ready
+  Start Time:             2023-08-22T22:48:02Z
+Events:
+  Type    Reason   Age   From               Message
+  ----    ------   ----  ----               -------
+  Normal  Created  36m   rayjob-controller  Created cluster rayjob-sample-raycluster-j6t8n
+  Normal  Created  32m   rayjob-controller  Created k8s job rayjob-sample
+```
+
+To view the logs, first find the name of the pod running the job with `kubectl get pods`.
+
+Sample output:
+
+```bash
+NAME                                        READY   STATUS      RESTARTS   AGE
+kuberay-operator-8b86754c-r4rc2             1/1     Running     0          25h
+rayjob-sample-raycluster-j6t8n-head-kx2gz   1/1     Running     0          35m
+rayjob-sample-w98c7                         0/1     Completed   0          30m
+```
+
+Here the Ray cluster is still running because we did not set `shutdownAfterJobFinishes` in the `RayJob` spec.  If you set `shutdownAfterJobFinishes` to `true`, the cluster will be shut down after the job finishes.
```

Comment:
```suggestion
Here the Ray cluster is still running because we did not set `shutdownAfterJobFinishes` in the `RayJob` spec. If you set `shutdownAfterJobFinishes` to `true`, the cluster will be shut down after the job finishes.
```